source/LogbookPanel.cpp

```diff
@@ -115,6 +120,15 @@ void LogbookPanel::Draw()
 	Point textOffset(0., .5 * (LINE_HEIGHT - font.Height()));
 	// Start at this point on the screen:
 	Point pos = Screen::TopLeft() + Point(PAD, PAD - categoryScroll);
+	// Draw the Reputations tab.
+	bool reputationsSelected = selectedName == "Reputations";
+	/*if(reputationsSelected)
+	{
+		FillShader::Fill(pos + highlightOffset - Point(1., 0.), highlightSize + Point(0., 2.), lineColor);
+		FillShader::Fill(pos + highlightOffset, highlightSize, backColor);
+	}
+	font.Draw("Reputations", pos + textOffset, bright);
+	pos.Y() += LINE_HEIGHT;*/
```

Comment:
do you need help with this maybe not working maybe?

Also if you still need time to finish things up you can set this PR as a draft